From 3e22463965f8b5db5229788223ddfb41388bbe3a Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Fri, 22 Mar 2019 10:42:33 +0100
Subject: [PATCH 19/27] common: Add getcap/setcap to PRODUCT_PACKAGES

* These are pretty useful for debugging config.fs
  changes without rebuilding full system/vendor image.

Change-Id: I68fdeb2dc7c9f225784b55425407657645e34ac8
---
 config/common.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 65b582da..1cf11742 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -181,11 +181,13 @@ PRODUCT_PACKAGES += \
     bash \
     bzip2 \
     curl \
+    getcap \
     htop \
     lib7z \
     libsepol \
     pigz \
     powertop \
+    setcap \
     unrar \
     unzip \
     vim \
-- 
2.17.1

